deploy-java/action.yml

```diff
@@ -127,6 +133,12 @@ runs:
         aws ecr put-image --region us-west-2 --repository-name ${{ inputs.appName }} --image-tag ${{ inputs.versionTag }} --image-manifest "$manifest"
         echo "restart=true" >> $GITHUB_OUTPUT
 
+    - name: Configure AWS Credentials
+      uses: aws-actions/configure-aws-credentials@v2
```

Comment:
4.0.1 is new standard?

Reply:
Seems that 
`uses: aws-actions/configure-aws-credentials@v2`
is what we are using elsewhere in this actions repo. I just copied and pasted. I will raise something about this in our cross team retro. We are quite sporadic with changing the version and it's very hard to know what the standard is at a given time if we don't update all repos in sync and seems like a quite a bit of work to be recording each different action version we use [separately](https://cartoncloud.atlassian.net/wiki/spaces/PD/pages/70681164/Components+and+Dependencies+-+Versions+in+Use+and+End-of-life+notes). The AWS credentials is not on that list. We could adopt one of the models:
- Every repo maintains separate and independent versions, in which case whoever maintains the repository is responsible and there is no global standard.
- We standardise our repos in which case when someone makes an update to an action version it should be rolled out everywhere.

At the moment we seem to sit somewhere in between, where we have a global standard but we are only incrementally roll it out and generally we are waiting for a developer to make changes in that repository before requiring the version to be upgraded at the same time. Which could be the way we want to do it, it's just that it requires very acute developers/reviewers to know global standards for all actions when they are coding and reviewing which I find quite tricky. Can also create inconsistencies in a single repository (for example if I update this then other actions in this repo will be using different versions. If I go through and update all of them then I should regression test each action I touched which seems unrelated to the ticket I'm working on).

Reply:
In saying that, since this is the way we currently do it, I have upgrade to version 4 for now as requested.